holidays/countries/china.py

```diff
@@ -326,51 +201,234 @@ class CHN(China):
 
 
 class ChinaStaticHolidays:
-    # Special Cases.
-
-    # New Year's Day.
-    new_years_day_overflow = tr("元旦")
+    # Date format (see strftime() Format Codes).
+    substituted_date_format = tr("%Y-%m-%d")
+    # Day off (substituted from %s).
+    substituted_label = tr("休息日（%s日起取代）")
 
-    # National Day.
-    national_day_2008_golden_week = tr("国庆节")
+    # Dragon Boat Festival.
+    dragon_boat_festival = tr("端午节")
 
     # Mid-Autumn Festival.
-    mid_autumn_festival_2010_special = tr("中秋节")
+    mid_autumn_festival = tr("中秋节")
+
+    # 70th Anniversary of the Victory of the Chinese People’s War of Resistance against
+    # Japanese Aggression and the World Anti-Fascist War.
+    victory_70_anniversary = tr("中国人民抗日战争暨世界反法西斯战争胜利70周年纪念日")
 
     special_public_holidays = {
+        2001: (
+            (JAN, 29, JAN, 20),  # spring_festival
```

Comment:
The comments look a bit unusual, could you make them look like "Labor day" or "Spring festival" for example?